packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -893,69 +907,181 @@ export class MergeTree {
 	 * @param segments - An array of (not necessarily contiguous) segments with increasing ordinals.
 	 */
 	private slideAckedRemovedSegmentReferences(segments: ISegment[]) {
+		function anyLocalRef(
```

Comment:
i would add some tests to merge-tree for backwards slide pref, ideally we shouldn't rely on sequence test coverage.